releases/v1.4.0-beta.toml

```diff
@@ -0,0 +1,92 @@
+# commit to be tagged for new release
+commit = "HEAD"
+
+project_name = "containerd"
+github_repo = "containerd/containerd"
+match_deps = "^github.com/(containerd/[a-zA-Z0-9-]+)$"
+
+# previous release
+previous = "v1.3.0"
+
+pre_release = true
+
+preface = """\
+The fifth major release of containerd includes a mix of new features and
```

Comment:
`s/major/minor/` 😅  (kidding, I'll show myself out)

Reply:
I think we have been pretty good about following semver principles, but their terminology does not apply well everywhere. Introducing a backwards incompatible version is definitely major and a very big deal, but that doesn't mean major features cannot be introduced in compatible ways. In our `RELEASES.md` we use the semver terminology, just not great wording in the release notes. If you really do think this confusing, we could use a synonym here, this is just the beta though so I might add some wording to clarify that anyway for now.

Reply:
We've done the same in previous releases, so I think it might be more confusing to change our terminology.

"Third feature release of containerd v1" also sounds awkward.

So yes, it's incorrect, but I'd list that under "creative use of the terminology" and don't bother changing